addons/account/models/account_move.py

```diff
@@ -2275,7 +2275,7 @@ def is_internal_partner(partner):
         values = {
             'name': self.default_get(['name'])['name'],
             'invoice_source_email': from_mail_addresses[0],
-            'partner_id': partners and partners[0].id or False,
+            'partner_id': partners and not custom_values.get('company_id') and partners[0].id or False,
```

Comment:
`(custom_values or {}).get('company_id')`
`custom_values` might be `None`

Reply:
```suggestion
            'partner_id': not custom_values.get('company_id') and partners[:1].id,
```

Reply:
Because the problem is related to the alias and custom values is indeed evaluated with the alias value. Having said that, I'm not sure if this is the best way to solve the problem, any kind of suggestion is much appreciated :)